src/api/CoinBase.ts

```diff
@@ -0,0 +1,26 @@
+import { CoinBaseResponse, SourceApi } from '../types/RelayPricer';
+import fetch, { Response } from 'node-fetch';
+
+export default class CoinBase implements SourceApi {
+    // we should get this address from config file or enviroments
+    private readonly URL: string = 'https://api.coinbase.com/v2/exchange-rates';
```

Comment:
just move it to a constant outside of the class for now, eg: `const COINBASE_URL  = ...`

Reply:
When you say this, you mean like to constant class?

Reply:
updated